### ISSUE TITLE
Build and Deploy Your SAP Fiori App to SAP Business Technology Platform

### ISSUE THREAD

**Issue by lauropr** [SAPDocuments/Issues#241]
Tutorials: https://developers.sap.com/tutorials/appstudio-fioriapps-mta-build-deploy.html
--------------------------

Hi! While being successfully able to deploy my Fiori app to BTP, when accessing the app's URL I get the following authorization issue:

"No client with requested id: sb-portal-cf-dt-approuter!t3664"

Authorization Request Error
There was an error. The request for authorization was invalid.

Any idea how to solve it? Is it related to lack of configuration of authorization in my dev instance?
Thanks!

**Comment by raz-korn** [comment]
Hi @lauropr ,

I think of the following:

1. Subscription to the launchpad service is missing.
2. When accessing the url, it needs to be modified from "cpp" to "launchpad". The needed change is described in the tutorial.

Regards - Raz

**Comment by lauropr** [comment]
Thanks Raz!

I've reviewed my subscriptions and can see the launchpad service under Instances and Subscriptions (I tried re-subscribing also).

If I change the URL from cpp to launchpad, I get a blank screen. Looking more closely on the browser's failed network calls, I see a 500 error:

Request URL: https://...trial.launchpad.cfapps.eu10.hana.ondemand.com/442aebcd-d01e-48b3-afeb-a60b31994c2b.NW.comznorthwindmtaznorthwindmta-1.0.0/resources/sap-ui-core.js
Request Method: GET
Status Code: 500 Internal Server Error

Looks like it's trying to access the ui5 core from my project - iautomatically set it in manifest.json so I'm confused as to why it would happen. 

Worth noting I've used the template generator for all pieces - MTA, App Router and UI5 application.
